bootstrap/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:buster
+COPY setup.py /
+COPY startup.json.default /
+COPY main.py /
+RUN python3 setup.py install && chmod +x /main.py
+ENTRYPOINT /main.py
```

Comment:
I'm not a huge fan to have all files under `/` this can result in a mess after some updates or new files, maybe creating your own `/bootstrap` or `/service` folder for organization ?

Reply:
sure, as long as we don't go to deep I'm ok with it.

Reply:
![image](https://user-images.githubusercontent.com/1215497/92767862-e78d3f80-f36d-11ea-94f2-5404a50bf395.png)